pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -674,10 +674,7 @@ self: super: {
   });
 
   nvim-teal-maker = super.nvim-teal-maker.overrideAttrs (old: {
-    postPatch = ''
-      substituteInPlace lua/tealmaker/init.lua \
-        --replace cyan ${luaPackages.cyan}/bin/cyan
-    '';
+    passthru.pathDependencies = [ luaPackages.cyan ];
```

Comment:
Can you elaborate on this change. How does it replicate the `substituteInPlace` from the previous diff?